softlayer/vm/softlayer_vm.go

```diff
@@ -28,6 +28,7 @@ const (
 	softLayerVMtag = "SoftLayerVM"
 	ROOT_USER_NAME = "root"
 	deleteVMLogTag = "DeleteVM"
+	ISCSIMANAGEMENT = "ISCSI Management"
```

Comment:
Also, if these are all log tags then follow the convention like: `softLayerVMtag ` etc